Makefile

```diff
@@ -415,7 +415,7 @@ docs:
 kind:
 	./contrib/localdev/bootstrap-kind-cluster.sh
 
-.PHONY: kind-install-tetragon 
+.PHONY: kind-install-tetragon
```

Comment:
Can you please split the patch into multiple patches so that there is one patch per change?
Having one patch per each change in the 1. to 4. list would make it much easier to review and manage the patches.

Reply:
Sure! I think I can also include the fix for the CRD version in the commit where I update CRD files.
I will update you once done:)

Reply:
I did this. I needed to push with force to rewrite the history, but hopefully, not it looks more readable :pray:

Reply:
Thanks!

When you want to re-request a review, please notify the reviewers by clicking on the github button "Re-request review".


![1](https://github.com/cilium/tetragon/assets/221481/b83f6f18-b2e9-4880-b487-6e5341f20c11)

Reply:
Oh, that's great, thank you! I will use this button from now on:)